src/components/ProAmm/ProAmmFee.tsx

```diff
@@ -26,6 +26,18 @@ import { useUserSlippageTolerance } from 'state/user/hooks'
 import { basisPointsToPercent, calculateGasMargin } from 'utils'
 import { unwrappedToken } from 'utils/wrappedCurrency'
 
+/**
+  800                   -> '800'
+  0.123123123           -> '0.12312312'
+  0.123                 -> '0.123'
+  0.12342224234234324   -> '0.12342224'
+
+  This function is to remove trailing 0's after using toFixed
+ */
+const formatUSDValue = (val: number) => {
```

Comment:
we already have `formatNumberWithPrecisionRange`

Reply:
Nice. This is exactly what I wanted